net-libs/tox/tox-9999.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit autotools eutils git-r3 user systemd
+
+DESCRIPTION="Encrypted P2P, messaging, and audio/video calling platform"
+HOMEPAGE="https://tox.chat"
+SRC_URI=""
+EGIT_REPO_URI="https://github.com/irungentoo/toxcore.git
+	git://github.com/irungentoo/toxcore.git"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS=""
+IUSE="+av daemon log log-error log-info log-trace log-warn +no-log ntox static-libs test"
+
+REQUIRED_USE="^^ ( no-log log log-trace log-info log-warn log-error )"
+
+RDEPEND="
+	>=dev-libs/libsodium-0.6.1[urandom,asm]
```

Comment:
(trivial) I suggest you keep USE deps sorted lexically, i.e. `[asm,urandom]`. Makes finding them easier.